package.xml

```diff
@@ -13,8 +13,8 @@
  <date>2018-01-19</date>
  <time>16:06:07</time>
  <version>
-  <release>1.19.0</release>
-  <api>1.19.0</api>
+  <release>1.19.1</release>
+  <api>1.19.1</api>
```

Comment:
@stanley-cheung do you know why we are updating both the api and release versions? If the api has not changed, we should not bump it up, right? Not a big issue for this PR though...

Reply:
Yea I will make a note of these and fix it in `master`. But this `api` version doesn't really matter.

Reply:
Does it mean it can be removed?

Reply:
That I am not sure, there's a format for these PECL extension.